templates/wafer.sponsors/packages.html

```diff
@@ -0,0 +1,30 @@
+{% extends "wafer/base.html" %}
+{% load i18n %}
+{% block content %}
+<section class="wafer wafer-sponsorship-packages">
+<h1>{% trans 'Sponsorship Packages' %}</h1>
+<p>Please mail the <a href="mailto:team@za.pycon.org">pyconza team</a> if you would like to sponsor or have any queries.</p>
+<div class="wafer list">
+    {% for package in sponsorshippackage_list %}
+    <section class="wafer wafer-sponsorhip-package">
```

Comment:
Shoudn't this be be "wafer-sponsorship-package" ?

(It's worth filing an isssue against wafer, which has the same typo)